code/online_challenges/src/project_euler/problem_025/problem_025.cpp

```diff
@@ -0,0 +1,38 @@
+#include <iostream>
+#include <vector>
+using namespace std;
+
+int main() 
+{
+    vector < int > temp, prevFibonacci, currFibonacci;
+
+    int count = 2;
+
+    prevFibonacci.push_back(1);
+    currFibonacci.push_back(1);
+        
+    while (currFibonacci.size() < 1000) 
+    {
+        int carry = 0;
+        count++;
+        for (int i = 0; i < prevFibonacci.size(); i++) 
+        {
+            temp.push_back((currFibonacci[i] + prevFibonacci[i] + carry) % 10);
```

Comment:
It doesn't really matter if a variable is declared inside a loop or not. I would suggest that you declare it inside the loop, since it is used temporarily. Read this [great answer](https://stackoverflow.com/a/7959658/6525260) for more info.